todos-app/src/DataStructure.ts

```diff
@@ -0,0 +1,9 @@
+export interface ITodo {
```

Comment:
I would refer to these as models rather than data structures. For small projects, they can all live in a single file like `models.ts` or `/models/Todos.ts`

Reply:
Fixed it. I replaced _DataStructure.ts_ by _/App/models/Todos.ts_